3-create_ros_2_package/README.md

```diff
@@ -0,0 +1,111 @@
+# Exercises 3 - Create ROS 2 package
+For the upcoming exercises, you will start writing some Python code.
+ROS 2 organizes code neatly in **packages** to separate different 
+functionalities, and make them to be easily installable and shareable.
+
+You can follow this tutorial to easily create a new package using 
+[Turtle Nest](https://github.com/Jannkar/turtle_nest) GUI tool, or learn how to create packages manually from the command line by following 
+[the official ROS 2 documentation](https://docs.ros.org/en/humble/Tutorials/Beginner-Client-Libraries/Creating-Your-First-ROS2-Package.html).
+
+## Create a new package
+1. Bring up the Docker container and open a new terminal inside it ([How to run Docker container](/0-setup/Docker%20Cheat%20Sheet.md))
```

Comment:
Changed the wording now to match the tutorial. But I wouldn't add the commands here, as we would otherwise also mention the xhost + and cd'in into the folder. So I think it is better to just link and not repeat.

Reply:
looks good now